actors/ppo.py

```diff
@@ -319,7 +319,12 @@ async def impl(self, actor_session):
         assert config.environment_specs.num_players == 1
 
         # Get model
-        serialized_model = await actor_session.model_registry.retrieve_model(config.model_id, config.model_iteration)
+        if config.model_iteration == -1:
+            serialized_model = await actor_session.model_registry.track_latest_model(config.model_id)
```

Comment:
in that case you can directly provide `PPOModel.deserialize_model` to the function so that the deserialized model is cached.

Reply:
Done